packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -26,21 +26,31 @@ import fetch from "cross-fetch";
 import safeStringify from "json-stringify-safe";
 
 import type { AxiosRequestConfig, RawAxiosRequestHeaders } from "./axios.cjs";
-import { RouterliciousErrorTypes, throwR11sNetworkError } from "./errorUtils.js";
+import {
+	RouterliciousErrorTypes,
+	throwR11sNetworkError,
+} from "./errorUtils.js";
 import { pkgVersion as driverVersion } from "./packageVersion.js";
-import { QueryStringType, RestWrapper } from "./restWrapperBase.js";
+import {
+	buildUrlWithQueryString,
+	type QueryStringType,
+} from "./queryStringUtils.js";
+import { RestWrapper } from "./restWrapperBase.js";
 import { ITokenProvider, ITokenResponse } from "./tokens.js";
 
 type AuthorizationHeaderGetter = (token: ITokenResponse) => string;
 export type TokenFetcher = (refresh?: boolean) => Promise<ITokenResponse>;
 
+const buildRequestUrl = (requestConfig: AxiosRequestConfig) =>
+	requestConfig.baseURL !== undefined
+		? `${requestConfig.baseURL ?? ""}${requestConfig.url ?? ""}`
+		: requestConfig.url ?? "";
+
 const axiosRequestConfigToFetchRequestConfig = (
 	requestConfig: AxiosRequestConfig,
 ): [RequestInfo, RequestInit] => {
-	const requestInfo: string =
-		requestConfig.baseURL !== undefined
-			? `${requestConfig.baseURL}${requestConfig.url ?? ""}`
-			: requestConfig.url ?? "";
+	const requestInfo: string = buildRequestUrl(requestConfig);
```

Comment:
Why do we call it `requestInfo` and not `requestUrl` while it is just holding a url.

Reply:
The Fetch API type for the first `fetch()` param is `RequestInfo` (i.e. `fetch(requestInfo: RequestInfo, requestInit: RequestInit)` so I think I named it `requestInfo` for that reason. I agree `const requestUrl: string` or `const requestInfo: RequestInfo` would probably be better, and since it's always URL, `requestUrl: string` makes sense